standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java

```diff
@@ -36,6 +36,13 @@
    * Stops the server
    */
   void stopPlatform();
+  
+  /**
+   * Server uptime.
+   * 
+   * @return the amount of milliseconds that have passed since the server was started
```

Comment:
Does started mean since the last state transition? What about zaps?

Reply:
this is purely server start regardless of server state.

Reply:
So if a server gets zapped it's uptime restarts... it kinda leaks the way zaps happen but I guess that is relatively harmless.